src/Controller/CRUDController.php

```diff
@@ -66,11 +68,29 @@ class CRUDController extends Controller
      */
     private $templateRegistry;
 
-    public function setContainer(?ContainerInterface $container = null): void
+    public function setContainer(ContainerInterface $container): ?ContainerInterface
     {
-        $this->container = $container;
+        $result = parent::setContainer($container);
 
         $this->configure();
+
+        return $result;
+    }
+
+    public static function getSubscribedServices()
+    {
+        return array_merge(parent::getSubscribedServices(), [
+            'sonata.admin.pool' => Pool::class,
+            'sonata.admin.breadcrumbs_builder' => BreadcrumbsBuilderInterface::class,
+            'sonata.admin.audit.manager' => AuditManagerInterface::class,
+            'sonata.admin.object.manipulator.acl.admin' => AdminObjectAclManipulator::class,
```

Comment:
This one does not want to work for some reason I can't understand.

Reply:
It worked adding an alias